GETTING_STARTED.md

```diff
@@ -348,6 +348,7 @@ create(:user, last_name: "Doe").email
 
 Transient Attributes
 --------------------
+Transient attributes behave like virtual attributes without being available on the model itself. This allows for more complex logic inside factories without altering the model for this cause.
```

Comment:
I like the idea of adding an intro here, but I am not sure if "virtual attributes" will be clear to everyone reading this. I also would like to avoid using "model" since factory_bot can be used for any type of Ruby object.

What do you think about:

```suggestion
Transient attributes are available within the factory definition, but are not set on the object being built. This allows for more complex logic inside factories.
```

Reply:
Understood! "model" is probably wrongly worded here. The part "are available within the factory definition, but are not set on the object being built." is intuitively not super clear for me (personally), as it describes the functionality implicitly. Maybe we can find something in between, like "Transient attributes are attributes available within the factory definition only...", making it clear that transient attributes are, in fact, attributes.
However, I do understand this is a superfluous definition of the feature, it would just help me better to understand what's going on.

Reply:
So taking your suggestion and combining it with my suggestion, that might look something like:

```suggestion
Transient attributes are attributes only available within the factory definition, and not set on the object being built. This allows for more complex logic inside factories.
```

I am fine with that as well!